third_party/amd/python/test/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption("--device", action="store", default="cuda")
+
+
+@pytest.fixture
+def device(request):
```

Comment:
@peterbell10 It turns out there was no fixture for the AMD backend, so I added it, just like we did for the XPU backend.